.tests/cypress/integration/cart/add-to-cart-bundle.js

```diff
@@ -0,0 +1,90 @@
+describe('Bundle product', function () {
+  before(() => {
+    // Hide premissions popup
+    cy.setCookie('permission-cookies', 'true')
+    cy.setCookie('permission-profiling', 'true')
+    cy.setCookie('mage-cache-sessid', 'true')
+    // Keep cookies beween tests
+    Cypress.Cookies.defaults({
+      whitelist: [
+        'frontend',
+        'X-Magento-Vary',
+        'permission-profiling',
+        'PHPSESSID',
+        'form_key'
+      ]
+    })
+  })
+
+  it('Visits product', () => {
+    cy.visit('https://alpaca-ce-solr-demo.snowdog.pro/sprite-yoga-companion-kit')
+    cy.get('.breadcrumbs__list').should('be.visible')
+  })
+
+  it('Check visiblility of content', () => {
+    cy.get('.product-view__main-details').should('be.visible')
+    cy.get('[data-ui-id=page-title-wrapper]').should('be.visible')
+    cy.get('.product-view__information').should('be.visible')
+    cy.get('.product-view__sku').should('be.visible')
+    cy.get('.price-box').should('be.visible')
+    cy.get('#product-options-wrapper').should('be.visible')
+    cy.get('.bundle-option').should('be.visible')
+    cy.get('.quantity-update').should('be.visible')
+    cy.get('#product-addtocart-button').should('be.visible')
+    cy.get('[data-testid=product-tab__title]').contains('Details').should('be.visible')
+    cy.get('[data-testid=product-tab__title]').contains('Reviews').should('be.visible')
+  })
+
+  it('Test product options', () => {
+    cy.get('#product-options-wrapper').should('be.visible')
+      .then((elements) => {
+        cy.log(elements.length)
```

Comment:
I can not make that assertion as in case where one of the product will be out of stock it can be invisible and it will not be a bug, just a real state of product.  This test log the length and the click in each, in cypress we will be albe to check if test could click is same number of elements we logged.